internal/app/go-filecoin/node/node.go

```diff
@@ -389,6 +390,21 @@ func (node *Node) SetupMining(ctx context.Context) error {
 	return nil
 }
 
+func registeredProofsFromSectorSize(ss abi.SectorSize) (registeredSealProof abi.RegisteredProof, registeredPoStProof abi.RegisteredProof, err error) {
+	switch ss {
+	case 1 << 35:
```

Comment:
Please add/use constants for these sizes (`pkg/constants`)

Reply:
done